src/tlo/methods/contraception.py

```diff
@@ -744,6 +744,7 @@ def get_item_code_for_each_contraceptive(self):
         get_items_from_pkg = self.sim.modules['HealthSystem'].get_item_codes_from_package_name
 
         _cons_codes = dict()
+        # items for each method that requires an HSI to switch to
```

Comment:
These comments are useful (and other typos picked-up appreciated), but in general we should try to make the changes in a PR specific to it's aim and as small as possible for that aim. (Comments and other things you want to add incidentally, can be put into a seperate "omnibus comment changes" PR.)

Reply:
I have reverted those commits and created separate PR as you suggested: #1105.

Reply:
Thanks for that!